lldb/test/API/functionalities/data-formatter/type-recognizer/value-object-cast/TestRecognizedValue.py

```diff
@@ -0,0 +1,514 @@
+"""
+Use lldb Python API to test ValueObject type recognizer
+"""
+
+
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class RecognizedValueTestCase(TestBase):
+    def setUp(self):
+        # Call super's setUp().
```

Comment:
We don't need to waste comments explaining Python syntax. Also, calls to base class are usually done via `super()`, and not via name of the base.

Reply:
We use TestBase instead of super() for code consistency and clarity for inexperienced python users.